app/build.gradle

```diff
@@ -152,36 +152,36 @@ sonarqube {
 dependencies {
 	implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
 
-	def dp3t_sdk_version = '2.0.0'
+	def dp3t_sdk_version = '2.0.1-dev-37'
 	devImplementation "org.dpppt:dp3t-sdk-android:$dp3t_sdk_version-calibration"
 	teschtImplementation "org.dpppt:dp3t-sdk-android:$dp3t_sdk_version"
 	abnahmeImplementation "org.dpppt:dp3t-sdk-android:$dp3t_sdk_version"
 	prodImplementation "org.dpppt:dp3t-sdk-android:$dp3t_sdk_version"
 	logImplementation "org.dpppt:dp3t-sdk-android:$dp3t_sdk_version"
 
-	implementation 'androidx.appcompat:appcompat:1.1.0'
-	implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-	implementation 'androidx.fragment:fragment:1.2.4'
+	implementation 'androidx.appcompat:appcompat:1.2.0'
+	implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+	implementation 'androidx.fragment:fragment:1.2.5'
 	implementation 'androidx.lifecycle:lifecycle-viewmodel:2.2.0'
 	implementation 'androidx.lifecycle:lifecycle-livedata:2.2.0'
 	implementation 'androidx.viewpager2:viewpager2:1.0.0'
-	implementation 'androidx.security:security-crypto:1.0.0-rc01'
-	implementation 'androidx.work:work-runtime:2.3.4'
+	implementation 'androidx.security:security-crypto:1.0.0-rc03'
+	implementation 'androidx.work:work-runtime:2.5.0-beta02'
```

Comment:
Do we want to take in a beta here?

Reply:
yes, this was explicitly recommended by Google ;-)